java/ci/build-in-docker.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
```

Comment:
Missing the copyright and license comment block

Reply:
Added the copyright.